nuxt.config.ts

```diff
@@ -30,8 +30,9 @@ export default defineNuxtConfig({
     'yandex-metrika-module-nuxt3'
   ],
   runtimeConfig: {
+    BASE_URL: process.env.BASE_URL,
```

Comment:
Maybe the camel case will be better here? Like `yandexMetrika`

```suggestion
    baseUrl: process.env.BASE_URL,
```

Reply:
For example: https://nuxtjs.org/docs/directory-structure/nuxt-config/#publicruntimeconfig